.gitlab-ci.yml

```diff
@@ -19,11 +19,15 @@ variables:
 
 .src_workflow:
   rules:
-    - if: '$FULL_BUILD != "ON"'
+    - if: '$FULL_BUILD != "ON" && $BENCHMARKS_BUILD != "ON"'
 
 .full_workflow:
   rules:
-    - if: '$FULL_BUILD == "ON"'
+    - if: '$FULL_BUILD == "ON" && $BENCHMARKS_BUILD != "ON"'
+
+.benchmarks_workflow:
+  rules:
+    - if: '$FULL_BUILD != "ON" && $BENCHMARKS_BUILD == "ON"'
```

Comment:
We could alternatively use some sort of `CI_WORKFLOW_TYPE` var which could be src, full, or benchmarks. That way we don't have to handle the two variables. I just don't have access to nightly build settings.

Reply:
You now can!